packages/core/src/lib/Hero/Hero.tsx

```diff
@@ -3,25 +3,30 @@ import { color } from '@web3uikit/styles';
 import styles from './Hero.styles';
 import { IHeroProps } from './types';
 
-const { ImageStyled, LeftContainerDiv, RightContainerDiv, SectionStyled } =
-    styles;
+const {
+    ImageStyled,
+    LeftContainerDiv,
+    RightContainerDiv,
+    SectionStyled,
+} = styles;
 
 const Hero: React.FC<IHeroProps> = ({
     align = 'center',
-    backgroundColor = `${color.gray30}`,
+    backgroundColor,
```

Comment:
these styles can also be set via customize prop, and if these default values are kept those styles are not set.

Instead, these default values are set directly in the styles file

for eg:
```
 background-color: ${(p) => p.backgroundColor ?? p.customize?.backgroundColor ?? color.gray30};
```